content/_data/changelogs/weekly.yml

```diff
@@ -12284,6 +12284,62 @@
     # pull: 5661 (PR title: Add missing `@since`)
     # pull: 5667 (PR title: Increase timeout for branch)
 
+  - version: '2.308'
+    date: 2021-08-24
+    changes:
+      - type: bug
+        category: bug
+        pull: 5670
+        issue: 66379
+        authors:
+        - jtnord
+        references:
+          - pull: 5670
+          - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.21
+            title: Winstone 5.21 changelog
+        message: |-
+          Entry 1: Use the JVM's default keystore type for the Jenkins server when terminating TLS connections within Jenkins (ie. Jenkins is started with the <code>--httpsPort</code> argument).
+      - type: rfe
+        category: rfe
+        pull: 5664
+        authors:
+        - janfaracik
+        message: |-
+          Update appearance for feed bar and description button to be modern and consistent.
+      - type: rfe
+        category: rfe
+        pull: 5615
+        issue: 66105
+        authors:
+        - i-am-vt
+        message: |-
+          Entry 1: Issue, When the **buildWithParameter** API is called, if the requests with the same parameters in the queue are merged, the http response code of the request uses a more appropriate 303(see other) instead of 201(created).
```

Comment:
```suggestion
          When the **buildWithParameter** API is called, if the requests with the same parameters in the queue are merged, the http response code of the request uses a more appropriate 303(see other) instead of 201(created).
```